### PR TITLE
fix monorepo dependencies

### PR DESCRIPTION
**Pull request by threepointone**
Fixes https://github.com/automerge/automerge-repo/issues/169

Monorepos are hard. One needs to keep dependency versions and ranges accurate across the repository, and keep checking as the codebase grows. In this PR, I used `@manypkg/cli` (a monorepo "linter") to check and fix versions across the repo, and applied changes as required.

Of note, this fixes the apps under `/examples`.

--- 

More notes inline in the PR. I think I tested the changes right, but please verify!

### CODE REVIEW

File: examples/automerge-repo-demo-counter-svelte/package.json [hunk 1]
```diff
@@ -15,15 +15,16 @@
     "@automerge/automerge-repo-network-websocket": "^1.0.3",
     "@automerge/automerge-repo-storage-indexeddb": "^1.0.3",
     "@automerge/automerge-repo-svelte-store": "^1.0.3",
-    "svelte": "^3.49.0"
+    "svelte": "^3.0.0"
```
Comment (threepointone) [inline examples/automerge-repo-demo-counter-svelte/package.json:18]: test with the lowest version specified as the peer dependency in `automerge-repo-svelte-store`

File: package.json [hunk 2]
```diff
@@ -15,12 +15,29 @@
     "dev": "run-p watch start:syncserver dev:demo",
     "pub": "lerna publish --yes",
     "start:syncserver": "cross-env DEBUG='WebsocketServer' yarn workspace @automerge/example-sync-server start",
+    "repocheck": "manypkg check",
```
Comment (threepointone) [inline package.json:18]: you can now run `yarn repocheck` in the root yourself to check whether there are any problem/drifts on dependency versions.

File: packages/automerge-repo-network-broadcastchannel/package.json [hunk 3]
```diff
@@ -29,5 +28,8 @@
   },
   "publishConfig": {
     "access": "public"
+  },
+  "devDependencies": {
```
Comment (threepointone) [inline packages/automerge-repo-network-broadcastchannel/package.json:32]: peerDependencies are also marked as devDependencies

**Comment by pvh** [comment]
All merged! Thanks @threepointone, `manychk` should come in handy in the future to avoid this mistake recurring.

**Pull request closed without merging**